python/lsst/ts/guitool/widget/control_tabs.py

```diff
@@ -0,0 +1,147 @@
+# This file is part of ts_guitool.
+#
+# Developed for the LSST Telescope and Site Systems.
```

Comment:
Please use the modern version with `Vera Rubin Observatory` instead of `LSST Telescope and Site` here and everywhere else.